lib/progress_bar.rb

```diff
@@ -96,6 +113,10 @@ def width_of(meter)
     send(:"#{meter}_width")
   end
 
+  def render_title
+    "[#{title}]" if title
```

Comment:
I'm not sure I like how this looks when rendered. What about having it bare? Or a colon suffix?

```
My Title [########     ][...]
Other Title: [######        ][...]
```

I also wish there was a way to automagically align everything when this happens, but I don't see how to accomplish that without a specific `:title_width` option, which I'm not fond of...